frontend/packages/process-editor/src/components/ConfigPanel/EditUniqueFromSignaturesInDataTypes/EditUniqueFromSignaturesInDataTypes.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { useBpmnContext } from '../../../contexts/BpmnContext';
+import { StudioProperty } from '@studio/components';
+import { useTranslation } from 'react-i18next';
+import { PersonPencilIcon } from '@studio/icons';
+import { SelectUniqueFromSignaturesInDataTypes } from './SelectUniqueFromSignaturesInDataTypes';
+import classes from './EditUniqueFromSignaturesInDataTypes.module.css';
+import { getSelectedDataTypes } from './UniqueFromSignaturesInDataTypesUtils';
+import { StudioModeler } from '../../../utils/bpmnModeler/StudioModeler';
+
+export const EditUniqueFromSignaturesInDataTypes = () => {
+  const { t } = useTranslation();
+  const { bpmnDetails } = useBpmnContext();
+  const selectedDataTypes = getSelectedDataTypes(bpmnDetails);
+  const existingDataTypeForTask = selectedDataTypes.length > 0;
+  const [isSelectVisible, setIsSelectVisible] = useState(false);
+
+  const studioModeler = new StudioModeler();
+  const tasks = studioModeler.getAllTasksByType('bpmn:Task');
+  const signingTasks = tasks
+    .filter((item) =>
+      selectedDataTypes.includes(
+        item.businessObject.extensionElements?.values[0]?.signatureConfig?.signatureDataType,
+      ),
+    )
+    .map((item) => ({
+      id: item.businessObject.extensionElements?.values[0]?.signatureConfig?.signatureDataType,
+      name: item.businessObject.name,
+    }));
```

Comment:
maybe the same destructure above can be used here?

Reply:
Done!